build.gradle

```diff
@@ -1,38 +1,33 @@
 buildscript {
 	repositories {
+		// External libs - jcenter is Bintray and is supposed to be a superset of Maven Central, but do both just in case
+		jcenter()
+		mavenCentral()
 		maven {
-			url "https://plugins.gradle.org/m2/"
+			url = uri("https://plugins.gradle.org/m2/")
 		}
 	}
+
 	dependencies {
-		classpath "gradle.plugin.net.minecrell:licenser:0.3"
+		//Spotbugs
+		classpath("gradle.plugin.com.github.spotbugs.snom:spotbugs-gradle-plugin:4.0.0")
+
+		// SonarQube / Cloud scanning
+		classpath("org.sonarsource.scanner.gradle:sonarqube-gradle-plugin:2.8")
 	}
 }
 
-apply plugin: 'java'
-apply plugin: 'idea'
-apply plugin: 'net.minecrell.licenser'
-
-repositories {
-	mavenCentral()
-	maven {
-	url "http://artifactory.terasology.org/artifactory/virtual-repo-live"
-	}
+plugins {
+	id("terasology-module")
```

Comment:
Negative - this build file does not serve as a Terasology module. It gets that from Jenkins / other automation. This build file should let it build entirely independently - also the reason why the repo has a Gradle wrapper etc :-)

Reply:
all we need is the kallist module right :?

Reply:
Come to think of it - we have a Kallisti both here under the Terasology org and under MovingBlocks. So we could just reduce this one to module-level (no `build.gradle` at all, uses JOML) and leave the other repo as-is in case of ever wanting to use Kallisti as a lib, but otherwise not mind it until then